### DISCUSSION TITLE
widevine dll

### DISCUSSION THREAD

**Discussion by vkonvict** [community/community#45950] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

sorry saw your thread about downloadloading widevine videos. i need the dlls to replace the dlls on mine but the folder you linked to has nothing now.
can you send them?

**Comment by queenofcorgis** [comment]
Hi @vkonvict ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Widevine. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
